modules/react/layout/lib/Box.tsx

```diff
@@ -100,39 +104,10 @@ export const boxStyleFn = <P extends BoxProps>(props: P) => {
 };
 
 // Meant to be used with elements. The `shouldForwardProps` will remove all style props
-const StyledBoxElement = styled('div', {shouldForwardProp})<StyledType & BoxProps>(
-  {
-    boxSizing: 'border-box',
-  },
-  border,
-  color,
-  depth,
-  flexItem,
-  gridItem,
-  font,
-  layout,
-  position,
-  space,
-  text
-);
+const StyledBoxElement = styled('div', {shouldForwardProp})<StyledType & BoxProps>(boxStyleFn);
```

Comment:
We had these three style prop lists that were error-prone to keep the same. So now we have one list in `boxStyleFn` and we're passing that to `StyledBoxElement` and `StyledBoxComponent`. That should make it easier to manage style prop lists.